main.css

```diff
@@ -0,0 +1,76 @@
+@import url(./reset.css);
+@import url(./components/components.css);
+
+/* html {
+    font-size: 62.5%;
+} */
+
+/* @media only screen and (max-width: 800px){
+	html {
+        font-size: 50%;
+    }
+} */
+
+body {
+    
+    background-color: white;
+}
+
+header {
+    background-color: var(--primary-color);
+    padding: 0.5rem; 
+    display: flex;
+    justify-content: space-between;    
+}
+
+.logo-img {
+    width: 10rem;
+    height: 4rem;
+    padding: -0.5rem;
+}
+
+
+nav, 
+.socials {
+    display: flex;
+    gap: 1rem;
+    font-size: 1.2rem;
+    align-items: center;
+}
+
+
+.hero-img {
+    width: 100vw;
+    height: auto;
+}
+
+
+main {
+    display: flex;
+    flex-direction: row;
+    font-size: 1.2rem;
+}
+
+aside {
+ background-color: var(--white);
+ /* width: 5rem; */
+ display: flex;
+ flex-direction: column;
+ justify-content: center;
+ gap: 1rem;
+ height: 100vh;
+ width: 25vw;
+ align-items: center;
+}
+
+section {
```

Comment:
same thing if you use section any where afterwords and you want diffrent style for that sestion then its difficult so use class so its best practice